### PR TITLE
Add deprecation notice to README.md

### PR DESCRIPTION
**Pull request by aserio**
Add deprecation notice to README.md

### CODE REVIEW

**Comment by aserio** [comment]
Hi @dgaliffiAMD, Adam Tran has asked that you review this PR.

File: README.md [hunk 1]
```diff
@@ -38,4 +43,4 @@ Please cite Caffe in your publications if it helps your research:
 
 ## hipCaffe: the HIP Port of Caffe
 
-For details on the HIP Port of Caffe, please take a look at the [README.ROCm.md](https://github.com/ROCmSoftwarePlatform/hipCaffe/blob/hip/README.ROCm.md) file.  
\ No newline at end of file
+For details on the HIP Port of Caffe, please take a look at the [README.ROCm.md](https://github.com/ROCmSoftwarePlatform/hipCaffe/blob/hip/README.ROCm.md) file.
```
Comment (dgaliffiAMD) [inline README.md]:
```suggestion
For details on the HIP Port of Caffe, please take a look at the [README.ROCm.md](https://github.com/ROCm/hipCaffe/blob/hip/README.ROCm.md) file.  
```
We might as well fix the link, if we're already touching the file.

**Review by dgaliffiAMD — CHANGES_REQUESTED** [review]
Just one little suggestion to also update a stale link. Looks good otherwise.

**Comment by aserio** [comment]
@dgaliffiAMD, I have accepted your changes.

**Pull request merged**